src/main/java/org/spongepowered/api/world/WorldTypes.java

```diff
@@ -40,6 +41,8 @@
 public final class WorldTypes {
 
     // @formatter:off
+    public static final Registry<WorldType> REGISTRY = WorldTypes.registry();
```

Comment:
If the game is capable of restarting the server (i.e. the client), this is going to hold a stale reference (or if called at the wrong time, cause the class to not load altogether).

I'm not sure why this needs to be a field. Unlike the other fields, this isn't a member of the world types (same thing for other classes). I think the registry access should be a method.